src/GPU_API/CUAPI_SendExtPotGREP2GPU.cu

```diff
@@ -0,0 +1,42 @@
+#include "CUAPI.h"
+#include "CUPOT.h"
+
+#if ( defined GPU  &&  defined GRAVITY )
+
+
+extern real   *h_ExtPotTable;
+extern double *h_GREP_Lv_Data_New;
+extern double *h_GREP_Lv_Radius_New;
+extern int     h_GREP_Lv_NBin_New;
+
+extern void CUAPI_SendExtPotTable2GPU( const real *h_Table );
```

Comment:
It is already in `Prototype.h`.

Reply:
Yes, but `Prototype.h` is not included in either `CUAPI.h` and `CUPOT.h`.